.github/workflows/upload-release.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we need to add the copyright header in the other workflows as well?

Reply:
yeah good point, actually this header should be referencing apache